vuu-ui/packages/vuu-table/src/useTableScroll.ts

```diff
@@ -264,13 +330,14 @@ export const useTableScroll = ({
             );
             if (direction && distance) {
               if (isVirtualScroll) {
-                console.log(
-                  `virtual scroll row required ${direction} ${distance} 
-                  first Row ${firstRowRef.current}`
-                );
-                // const from = firstRowRef.current + 1;
-                // console.log(`setRange from ${from}`);
-                // setRange({ from, to: from + viewportRowCount + 1 });
+                const offset = direction === "down" ? 1 : -1;
+                onVerticalScrollInSitu?.(offset);
+                const firstRow = firstRowRef.current + offset;
+                firstRowRef.current = firstRow;
+                setRange({
+                  from: firstRow,
+                  to: firstRow + viewportRowCount + 1,
```

Comment:
In other places, we're not adjusting index by `+ 1` for `to:` when setting range. Is this intentional here?

Reply:
This used to be the norm, but I realised when writing tests that the +1 was not correct. This is a leftover.